cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature

```diff
@@ -0,0 +1,64 @@
+Feature: IntegratedPlayer
+
+    @initialization
+    Scenario: Launch FCA for 'IntegratedPlayer'
+        Given the environment has been set up for 'IntegratedPlayer' tests
+        And 3rd party 'certification' app is launched with 'inactive' state
+
+    @sdk @IntegratedPlayer
+    Scenario: Positive Scenario : Create and validate broadcast player instance
+        When 1st party app invokes the 'Firebolt' API to 'create broadcastPlayer instance'
+        Then 'Firebolt' platform responds to '1st party app' with 'broadcastPlayer instance'
+        When 1st party app invokes the 'Firebolt' API to 'get status of broadcastPlayer'
+        Then 'Firebolt' platform responds to '1st party app' with 'status of broadcastPlayer'
+        When 1st party app invokes the 'Firebolt' API to 'load player'
+        Then 'Firebolt' platform responds to '1st party app' with 'value for player load'
+        When 1st party app invokes the 'Firebolt' API to 'play player'
+        Then 'Firebolt' platform responds to '1st party app' with 'value for player play'
+        When 1st party app invokes the 'Firebolt' API to 'get player tracks'
+        Then 'Firebolt' platform responds to '1st party app' with 'value for player tracks'
+        When 1st party app invokes the 'Firebolt' API to 'set audio track'
+        Then 'Firebolt' platform responds to '1st party app' with 'value for set audio track'
+        When 1st party app invokes the 'Firebolt' API to 'set captions track'
+        Then 'Firebolt' platform responds to '1st party app' with 'value for set captions track'
+        When 1st party app invokes the 'Firebolt' API to 'set position'
+        Then 'Firebolt' platform responds to '1st party app' with 'value for set position'
+        When 1st party app invokes the 'Firebolt' API to 'set speed'
+        Then 'Firebolt' platform responds to '1st party app' with 'value for set speed'
+        When 1st party app invokes the 'Firebolt' API to 'provide status'
+        Then 'Firebolt' platform responds to '1st party app' with 'value for provide status'
+        When 1st party app invokes the 'Firebolt' API to 'get player status'
+        Then 'Firebolt' platform responds to '1st party app' with 'value for player status'
+        When 1st party app invokes the 'Firebolt' API to 'get broadcast status of player'
+        Then 'Firebolt' platform responds to '1st party app' with 'broadcast status of player'
+        When 1st party app invokes the 'Firebolt' API to 'stop player'
+        Then 'Firebolt' platform responds to '1st party app' with 'value for player stop'
+        When 1st party app invokes the 'Firebolt' API to 'destroy player'
+        Then 'Firebolt' platform responds to '1st party app' with 'value for player destroy'
+
```

Comment:
also, regarding the 3rd flow , check with kevin on the event name
also the difference in methods for streaming player and broadcast player,  mention that to kevin. Maybe it was a miss or it was intentional